sycl/doc/extensions/experimental/sycl_ext_oneapi_non_uniform_groups.asciidoc

```diff
@@ -540,28 +541,28 @@ _Returns_: A linearized version of the `id` returned by `get_group_id()`.
 ----
 id_type get_local_linear_id() const;
 ----
-_Returns_: A linearized version of the `id` returned by `get_local_linear_id()`.
+_Returns_: A linearized version of the `id` returned by `get_local_id()`.
 
 [source,c++]
 ----
-range_type get_group_linear_range() const;
+linear_id_type get_group_linear_range() const;
 ----
-_Returns_: A linearized version of the `id` returned by `get_group_range()`.
+_Returns_: A linearized version of the `range` returned by `get_group_range()`.
 
 [source,c++]
 ----
-range_type get_local_linear_range() const;
+linear_id_type get_local_linear_range() const;
 ----
-_Returns_: A linearized version of the `id` returned by `get_local_range()`.
+_Returns_: A linearized version of the `range` returned by `get_local_range()`.
 
 [source,c++]
 ----
 bool leader() const;
 ----
-_Returns_: `true` for exactly one work-item in the fixed-size-group, if the calling
-work-item is the leader of the fixed-size-group, and `false` for all other
-work-items in the fixed-size-group. The leader of the fixed-size-group is guaranteed
-to be the work-item for which `get_local_id()` returns 0.
+_Returns_: `true` for exactly one work-item in the fixed-size-group, if the
```

Comment:
Note for reviewers: no real changes here, just a re-formatting to fit into 80 symbol limit per line